cherrypy/test/test_patch_request.py

```diff
@@ -0,0 +1,56 @@
+"""Simple test for HTTP PATCH requests."""
+
+import cherrypy
+from cherrypy.test import helper
+from pprint import pprint
+
+class PatchRequestTests(helper.CPWebCase):
+
+    @staticmethod
+    def setup_server():
+        """Sets up a basic cherryPy server for testing.
+        """
+
+        class Root(object):
+
+            @cherrypy.expose
+            def patch(self, param):
+                """Makes sure that PATCH requests work
+
+                Args:
+                    kwargs (dict): values passed in the URI and body
+
+                Returns:
+                    str: the values from the URI and body
+                """
+                print('[LOG] in PATCH method')
```

Comment:
NO PRINTS! Logging can be implented with `logging` module, but only if really needed.